### PR TITLE
Fix possible exception while importing playlists

### PR DESCRIPTION
**Pull request by orontee**
Refs: #7

### CODE REVIEW

File: mopidy_listenbrainz/listenbrainz.py [hunk 1]
```diff
@@ -232,15 +232,22 @@ def _collect_playlist_data(
             return None
         track_mbids = []
         for track_dto in dto.get("track", []):  # not tracks!
```
Comment (suaviloquence) [inline mopidy_listenbrainz/listenbrainz.py:234]: should the default be a dict `{}`? we try to get named keys from it
```suggestion
        for track_dto in dto.get("track", {}):  # not tracks!
```

**Review by suaviloquence — APPROVED** [review]
Awesome, thanks! :rocket:

File: mopidy_listenbrainz/listenbrainz.py (hunk 1 above)
Comment (suaviloquence) [reply mopidy_listenbrainz/listenbrainz.py:234]: I dont think this was changed in the PR so it's not a problem here but lmk if this should be changed.

Comment (orontee) [reply mopidy_listenbrainz/listenbrainz.py:234]: Good point!

Comment (suaviloquence) [reply mopidy_listenbrainz/listenbrainz.py:234]: note for self: fixed in #11

**Pull request merged**